CMakeLists.txt

```diff
@@ -1346,7 +1346,7 @@ if(MSVC)
     FILES $<TARGET_PDB_FILE:mixxx>
     CONFIGURATIONS Debug RelWithDebInfo
     DESTINATION "${MIXXX_INSTALL_BINDIR}"
-    COMPONENT "Debug Information (PDB)"
+    COMPONENT PBD_Debug_Information
```

Comment:
Typo. Also, the underscores look weird, let's just use PDB then.

Reply:
https://github.com/mixxxdj/mixxx/commit/6c6330cb56a4d38fcb76483229d0c1e9b480ae16